### PR TITLE
Implement setPassenger and other

### PR DESCRIPTION
**Pull request by kamcio96**

### CODE REVIEW

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 1]
```diff
@@ -767,12 +785,41 @@ public boolean isCustomNameVisible() {
 
     @Override
     public Entity getPassenger() {
-        throw new UnsupportedOperationException("Not supported yet.");
+        return passanger;
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: passenger

File: src/main/java/net/glowstone/entity/GlowEntity.java [hunk 2]
```diff
@@ -145,6 +146,17 @@ protected String disambiguate(Entity subject, String metadataKey) {
     private int fireTicks = 0;
 
     /**
+     * Passanger
+     */
+    private GlowEntity passanger;
```
Comment (mastercoms) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: passenger

**Comment by mastercoms** [comment]
I'm still really busy, so I won't be able to review any code until tonight or tomorrow night, so just spelling and formatting checks for now, sorry.

File: src/main/java/net/glowstone/entity/GlowEntity.java (hunk 2 above)
Comment (kamcio96) [inline src/main/java/net/glowstone/entity/GlowEntity.java]: Ohhh, really...
I checked whole code few times...

**Comment by kamcio96** [comment]
Np ;)

**Pull request merged**